### PR TITLE
Enhance instructions for contributors

### PR DESCRIPTION
**Pull request by akien-mga**

### CODE REVIEW

**Comment by akien-mga** [comment]
I've made it a bit more verbose, but I think it's still OK. We could also add some kind of TL;DR without details for the very lazy bug reporters :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,37 +1,59 @@
...
+* We can't take the time to look at your project, understand how it is set up and then figure out why it's failing.
  
-To speed up our work, prepare for us **a simple project** that isolates and reproduces the issue. This is always the **the best way for us to fix it**.
+To speed up our work, please prepare for us **a simple project** that isolates and reproduces the issue. This is always the **the best way for us to fix it**. You can attach a zip file with the minimal project directly to the bug report, by drag and dropping the file in the GitHub edition field.
 
-### Contributing Pull Requests
+## Contributing pull requests
 
-If you are adding new engine funcitonality, please make sure that:
+If you want to new engine functionalities, please make sure that:
```
Comment (29jm) [inline CONTRIBUTING.md]: This is a typo, it should be "If you want to add..."

Comment (akien-mga) [inline CONTRIBUTING.md]: Thanks, I fixed it :)

**Comment by vnen** [comment]
Cool :smiley:

**Pull request merged**